auto_nag/bugbug_utils.py

```diff
@@ -34,7 +36,11 @@ def retrieve_model(self):
             old_etag = None
 
         if old_etag != new_etag:
-            urlretrieve(model_url, f'{file_path}.xz')
+            try:
+                urlretrieve(model_url, f'{file_path}.xz')
+            except HTTPError:
+                logger.exception('Tool {}'.format(self.name()))
```

Comment:
Nit: make the logged error a bit more verbose

Reply:
with logger.exception, the exception is printed and we'll receive it in the mail.
Is it enough for you ?